samples/cpp/classification_sample_async/main.cpp

```diff
@@ -40,22 +40,22 @@ using namespace ov::preprocess;
  * @param argv list of input arguments
  * @return bool status true(Success) or false(Fail)
  */
-bool ParseAndCheckCommandLine(int argc, char* argv[]) {
+bool parse_and_check_command_line(int argc, char* argv[]) {
     gflags::ParseCommandLineNonHelpFlags(&argc, &argv, true);
     if (FLAGS_h) {
-        showUsage();
+        show_usage();
         showAvailableDevices();
```

Comment:
mixed coding style.

The same note is about:
- `parseInputFilesArguments()`
- `printInputAndOutputsInfo()`

---

It makes sense to completely resolve problem by PR. Not introduce a new one.

Reply:
I fixed main notes of NCC tool for C++ samples. In next PR we will add automatically tracking the regressions and resolve other notes